gunicorn/config.py

```diff
@@ -346,6 +346,15 @@ def validate_pos_int(val):
     return val
 
 
+def validate_float(val):
+    if not val:
+        return 0.0
```

Comment:
this would make the default to 0, where the current behaviour expect `0.1 * random.random()` .  There should be a decent default to have all the workers accepting  simultaneously. maybe `0.05` if we don't want to change the function. Thoughts?

Reply:
should work indeed. rethinking of it we need to keep the current default anyway, as some app (like one i have around is expecting it :)